### DISCUSSION TITLE
Inverse color scheme

### DISCUSSION THREAD

**Discussion by jakoblistabarth** [observablehq/plot#2224] [Q&A] [answered]
I would like to change the direction of a color scheme in observable plot. The code I came up with so far seems not very *plot-like*. Plus, it has the disadvantage that by setting the `range` explicitly the `n` property of the `color` scale does no longer work. I am sure there's a smarter way, but I could not wrap my head around it – What would be a better way to do that?

See [this notebook](https://observablehq.com/d/db87a3fe8863a02a) for a minimal reproducible example.

Thank you 🙂

**Comment by Fil** [comment, marked as answer]
Did you try to add `reverse: true` to the color scale options?

**Reply by jakoblistabarth** [reply]
Thank you! I am embarrassed that I've missed this in the docs 😳 . I was only looking for `inverse` in the docs – my bad!
